dashboard/app/controllers/application_controller.rb

```diff
@@ -323,6 +326,28 @@ def clear_sign_up_session_vars
     end
   end
 
+  # Check that the user is compliant with the Child Account Policy. If they
+  # are not compliant, then we need to send them to the lockout page.
+  def assert_child_account_policy
+    # Check that the child account policy is currently enabled.
+    return unless Cpa.cpa_experience(request)
```

Comment:
You are going to have the same error. Try to call it from the main context:
```suggestion
    return unless ::Cpa.cpa_experience(request)
```

Reply:
sounds good